src/Cohesity.Powershell/Scripts/ProtectionJobRun/Update-CohesityProtectionJobRun.ps1

```diff
@@ -0,0 +1,111 @@
+#### USAGE ####
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -JobRunId 65675,65163 -ExtendRetention 10
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -JobRunId 65675 -ExtendRetention 10
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -ExtendRetention 10
```

Comment:
worth mentioning this is in days , also negative reduces the retention by that many days and setting into 0 deletes the snapshot.

Reply:
Fixed, added comments in the source header